content/blog/pkgdown-2-0-0/index.Rmd

```diff
@@ -0,0 +1,195 @@
+---
+output: hugodown::hugo_document
+
+slug: pkgdown-2-0-0
+title: pkgdown 2.0.0
+date: 2021-11-29
+author: Hadley Wickham
+description: >
+    usethis 2.0.0 includes a major refresh of the default template (now 
+    using bootstrap 5), many new ways to customise your site, improvements
+    to code styling, and much, much, more.
+    
+photo:
+  url: https://unsplash.com/photos/1HIKnKtXEU0
+  author: Edgar Soto
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [pkgdown, devtools]
+---
+
+```{=html}
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+```
+We're very chuffed to announce the release of [pkgdown](https://pkgdown.r-lib.org) 2.0.0.
+pkgdown is designed to make it quick and easy to build a website for your package.
+Install it with:
+
+```{r, eval = FALSE}
+install.packages("pkgdown")
+```
+
+This is a massive release that brings a major visual refresh and a huge number of improvements.
+This release would not have been possible without pkgdown's newest author [Maëlle Salmon](https://masalmon.eu), who was the powerhouse behind many of the improvements in this release, particularly the switch to bootstrap 5, improved customisation, and implementation of local search.
```

Comment:
```suggestion
This release would not have been possible without pkgdown's newest author, [Maëlle Salmon](https://masalmon.eu), who was the powerhouse behind many of the improvements in this release, particularly the switch to Bootstrap 5, improved customisation, and implementation of local search.
```

Reply:
Also, we have bootstrap 5 here and Bootstrap 5 below, so it should just be one or the other. (I went uppercase above)